test/stdcpp/Makefile

```diff
@@ -0,0 +1,18 @@
+include ../common.mak
+
+TESTS:=array
+
+.PHONY: all clean
+all: $(addprefix $(ROOT)/,$(addsuffix .done,$(TESTS)))
+
+$(ROOT)/test_stdcpp.done: $(ROOT)/%.done : $(ROOT)/%
```

Comment:
This should just be
```
$(ROOT)/%.done : $(ROOT)/%
```

Reply:
I copied this from neighbour makefiles, which are exactly like this...

Reply:
But they'd have a test target `test_stdcpp` in that case.